### ISSUE TITLE
Wrong suggestion

### ISSUE THREAD

**Issue by GF-Huang** [microsoft/pylance-release#599]
## Environment data

-   VS Code version: 
```
版本: 1.51.0 (user setup)
提交: fcac248b077b55bae4ba5bab613fd6e9156c2f0c
日期: 2020-11-05T18:18:23.642Z
Electron: 9.3.3
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Windows_NT x64 10.0.18363
```

-   Extension version (available under the Extensions sidebar): `v2020.10.332292344`
-   OS and version: `Win10 1909`
-   Python version (& distribution if applicable, e.g. Anaconda): `3.7.6 64-bite (conda)`
-   Type of virtual environment used (N/A | venv | virtualenv | conda | ...): `conda`
-   Relevant/affected Python packages and their versions: XXX
-   Relevant/affected Python-related VS Code extensions and their versions: `Pylance v2020.11.0`
-   Value of the `python.languageServer` setting: `Pylance`

[**NOTE**: If you suspect that your issue is related to the Microsoft Python Language Server (`python.languageServer: 'Microsoft'`), please download our new language server [Pylance](https://marketplace.visualstudio.com/items?itemName=ms-python.vscode-pylance) from the VS Code marketplace to see if that fixes your issue]

## Expected behaviour

Input `keras.layers.LST` show me the `LSTM` suggestion.

## Actual behaviour

Show me what the shit

![image](https://user-images.githubusercontent.com/4510984/98746542-a9a1ab80-23f0-11eb-8a8c-bf95fb1feb38.png)

## Steps to reproduce:

[**NOTE**: Self-contained, minimal reproducing code samples are **extremely** helpful and will expedite addressing your issue]

I dont know how to reproduce, sometime occurs.

## Logs

<details>

<summary>Output for <code>Python</code> in the <code>Output</code> panel (<code>View</code>→<code>Output</code>, change the drop-down the upper-right of the <code>Output</code> panel to <code>Python</code>)
</summary>

<p>

```
XXX
```

</p>
</details>

**Comment by bschnurr** [comment]
Can you try non-anaconda version 2.4.0 of keras or newer from pip ?

It looks like the Keras team removed the legacy module decorators, that is causing this issue, as of version 2.4.0. (keras-team/keras/releases)
see issue #99

**Comment by savannahostrowski** [comment]
This issue has been opened for > 30 days with a waiting for information label so I'm going to close this. If this is still an issue, feel free to ping us and we can reopen.
